website/docs/r/polling_source.html.markdown

```diff
@@ -2,7 +2,7 @@
 layout: "sumologic"
 page_title: "SumoLogic: sumologic_polling_source"
 description: |-
-  Provides a Sumologic Polling source. This source is used to import data from various AWS products.
+  Provides a Sumologic Polling source. This source is used to import data from various AWS products as well as Azure Event Hub.
```

Comment:
I know this is the first Azure product, but from doc perspective maybe this is cleaner? What do you think?
```suggestion
  Provides a Sumologic Polling source. This source is used to import data from various AWS and Azure products.
```

Reply:
Sounds reasonable.